tests/phpunit/tests/block-bindings/postMetaSource.php

```diff
@@ -266,4 +275,35 @@ public function test_custom_field_with_unsafe_html_is_sanitized() {
 			'The post content should not include the script tag.'
 		);
 	}
+
+	/**
+	 * Tests that filter `block_bindings_source_value` is applied.
+	 *
+	 * @ticket 61181
+	 */
+
+	public function test_filter_block_bindings_source_value() {
```

Comment:
Nit: the empty line should get removed.
```suggestion
	public function test_filter_block_bindings_source_value() {
```

Reply:
By the way, I think this test should be general purpose in the first place as it isn't tied to post meta, so maybe `render.php` in the same folder would be a good fit. Having, two tests could work, too.

Reply:
I'll move it to `render.php`